samples/OcfServer.js

```diff
@@ -1,6 +1,24 @@
 // Copyright (c) 2017, Intel Corporation.
 
-var server = require('ocf').server;
+var ocf = require('ocf');
+var server = ocf.server;
+
+ocf.device = {
+	name: "MyOCFServer",
```

Comment:
Did we ever standardized the use of single quote or double quotes?  Might be good idea to keep them consistent.